### ISSUE TITLE
[Linux] `dnvm list` should display installed runtimes in a reasonable order

### ISSUE THREAD

**Issue by dougbu** [aspnet/dnvm#334]
Example scrambled output of `dnvm list` on Linux:
```
Active Version              Runtime Arch Location             Alias
------ -------              ------- ---- --------             -----
  *    1.0.0-beta6-12108    mono         ~/.dnx/runtimes      default
       1.0.0-beta6-11944    mono         ~/.dnx/runtimes
       1.0.0-beta5          coreclr x64  ~/.dnx/runtimes
       1.0.0-beta5-12103    coreclr x64  ~/.dnx/runtimes
       1.0.0-beta6-11941    mono         ~/.dnx/runtimes
       1.0.0-beta5          mono         ~/.dnx/runtimes
       1.0.0-beta4          mono         ~/.dnx/runtimes
       1.0.0-beta6-12108    coreclr x64  ~/.dnx/runtimes
       1.0.0-beta5-12103    mono         ~/.dnx/runtimes
       1.0.0-beta6-11923    mono         ~/.dnx/runtimes
```

**Comment by dougbu** [comment]
Bonus points to match order used on Windows :smile_cat:

**Comment by smoak** [comment]
What is the desired output? Just that the versions are sorted? What is the output on Windows (I don't have a Windows box)?

**Comment by mholo65** [comment]
Checked from @DamianEdwards presentation at NDC (https://vimeo.com/131191236#t=12m45s) and it's alphabetical ascending. Column order is Version, Runtime, Arch...

**Comment by BrennanConroy** [comment]
ae0bcc6e87d4a5550bac57d38aff25cdb9630f7c

**Comment by dougbu** [comment]
List is again unsorted on Linux, at least on a Mint 17.2 / Ubuntu 14.04 / Linux kernel 3.16 system. Example output:
```
$ dnvm list

Active Version              Runtime Arch OperatingSystem Alias
------ -------              ------- ---- --------------- -----
       1.0.0-beta8-15529    mono         linux/darwin    
  *    1.0.0-beta7          coreclr x64  linux           default
       1.0.0-beta8-15519    coreclr x64  linux           
       1.0.0-beta7          mono         linux/darwin    
       1.0.0-beta8-15530    mono         linux/darwin    
       1.0.0-beta8-15120    mono         linux/darwin    
       1.0.0-beta8-15530    coreclr x64  linux           
       1.0.0-beta6          mono         linux/darwin    
```

Have updated using `dnvm.sh update-self && . ~/.dnx/dnvm/dnvm.sh` to 1.0.0-beta8-15502. Script contains
```
for f in `echo $runtimes | sort -t. -k2 -k3 -k4 -k1`; do
```
but the `sort` isn't working.

**Comment by mholo65** [comment]
Fixed in #408 Just needed to replace whitespace with newline in order make sort working again.

**Comment by mholo65** [comment]
Fixed in Global install (fc63c6c), please close.
